static/linux/postrm

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+set -e
+rm /usr/bin/webtorrent-desktop
```

Comment:
Should this also remove `/opt/webtorrent-desktop`? Or is that done automatically by the OS?